models/stg_recharge__charge_line_item.sql

```diff
@@ -25,18 +25,18 @@ final as (
         subscription_id,
         vendor,
         title,
-        property_title as title_v2,
+        property_title as title_v2, 
         variant_title,
         sku,
         grams,
         quantity,
         price,
         shopify_product_id,
         shopify_variant_id,
-        property_charge_interval_frequency as charge_interval_frequency,
-        property_charge_interval_unit_type as charge_interval_unit,
-        property_shipping_interval_frequency as shipping_interval_frequency,
-        property_shipping_interval_unit_type as shipping_interval_unit,
+        property_charge_interval_frequency as charge_interval_frequency, 
+        property_charge_interval_unit_type as charge_interval_unit, 
+        property_shipping_interval_frequency as shipping_interval_frequency, 
+        property_shipping_interval_unit_type as shipping_interval_unit, 
```

Comment:
Same comment about these all being custom fields. We should let the variables handle these.

Reply:
Removed.